community_tasks/swiss_legal_evals.py

```diff
@@ -0,0 +1,402 @@
+`LightevalTaskConfig` class with its respective prompt function. The tasks 
+cover a variety of benchmarks, including: translation of laws, court decisions 
+and press releases.
+
+Author: Joel Niklaus
+"""
+from lighteval.logging.hierarchical_logger import hlog_warn
+import nltk
+
+nltk.download("punkt_tab")
+
+
+import numpy as np
+import re
+from dataclasses import dataclass
+
+
+from transformers import AutoTokenizer, AutoModelForSequenceClassification
+from lighteval.tasks.lighteval_task import LightevalTaskConfig
+from lighteval.tasks.requests import Doc
+from lighteval.metrics.metrics import Metrics
+from lighteval.metrics.metrics_sample import JudgeLLMMixEval, BertScore
+from lighteval.metrics.utils.metric_utils import (
+    MetricCategory,
+    MetricUseCase,
+    SampleLevelMetric,
+    SampleLevelMetricGrouping,
+)
+from lighteval.metrics.imports.bert_scorer import BERTScorer
+from lighteval.metrics.normalizations import remove_braces, remove_braces_and_strip
+from lighteval.tasks.extended.mix_eval.judge_prompts import (
+    flow_judge_for_freeform_template,
+    gpt_judge_for_closeended_freeform,
+)
+
+
+# CUSTOM METRICS
+def process_judge_response(x):
+    search = re.search(r"<score>\s(\d)\s</score>", x)
+    return int(search.group(1)) if search else 0
+
+
+def process_judge_response_freeform_gpt(x):
+    search = re.search(r"\[\[(\d.\d)\]\]", x)
+    answer = float(search.group(1) if search else 0)
+    return answer
+
+
+def freeform_flow_judge():
+    return SampleLevelMetricGrouping(
+        metric_name=["llm_judge_mixeval_flow"],
+        higher_is_better={"judge_score_flow": True},
+        category=MetricCategory.LLM_AS_JUDGE,
+        use_case=MetricUseCase.SUMMARIZATION,
+        sample_level_fn=JudgeLLMMixEval(
+            judge_model_name="flowaicom/Flow-Judge-v0.1",
+            template=flow_judge_for_freeform_template,
+            process_judge_response=process_judge_response,
+            judge_backend="vllm",
+            short_judge_name="flow",
+        ).compute,
+        corpus_level_fn={
+            "judge_score_flow": np.mean,
+        },
+    )
+
+
+def freeform_gpt_judge(judge_model_name: str = "gpt-4o"):
+    return SampleLevelMetricGrouping(
+        metric_name=[f"llm_judge_mixeval_{judge_model_name}"],
+        higher_is_better={"judge_score_{judge_model_name}": True},
+        category=MetricCategory.LLM_AS_JUDGE,
+        use_case=MetricUseCase.SUMMARIZATION,
+        sample_level_fn=JudgeLLMMixEval(
+            judge_model_name=judge_model_name,
+            template=gpt_judge_for_closeended_freeform,
+            process_judge_response=process_judge_response_freeform_gpt,
+            judge_backend="openai",
+            short_judge_name=judge_model_name,
+        ).compute,
+        corpus_level_fn={
+            f"judge_score_{judge_model_name}": np.mean,
+        },
+    )
+
+
+def bert_score(model_type: str = "xlm-roberta-large"):
```

Comment:
Why do you need to redefine the bert_scorer? Because you're using a different model?

Reply:
Yes, exactly. The main bert_scorer is not sufficiently configurable.